Makefile

```diff
@@ -1,6 +1,6 @@
 NAME = herokuish
 HARDWARE = $(shell uname -m)
-VERSION ?= 0.3.32
+VERSION ?= 0.3.35
```

Comment:
don't make this change yet. we handle that on release

Reply:
I'll revert it, assumed it was needed because of the version defined in the Dockerfile

Reply:
`Dockerfile` is primarily used on push to `master` and `release` by Docker Hub to build the image for distribution. The new versioned artifact is created only on a successful CI run on the `release` branch.